### ISSUE TITLE
"AssignTargetFramework" task could not be loaded from the assembly NuProj.Tasks.dll

### ISSUE THREAD

**Issue by ravensorb** [nuproj/nuproj#205]
I just started a brand new project and am running into an odd issue -- when I try to build the Nupkg file I get the following error 

c:\Projects\MyProject\packages\NuProj.0.10.4-beta-gf7fc34e7d8\tools\NuProj.targets(486,9): error MSB4062: The "AssignTargetFramework" task could not be loaded from the assembly c:\Projects\MyProject\packages\NuProj.0.10.4-beta-gf7fc34e7d8\tools\NuProj.Tasks.dll. Could not load file or assembly 'file:///c:\Projects\MyProject\packages\NuProj.0.10.4-beta-gf7fc34e7d8\tools\NuProj.Tasks.dll' or one of its dependencies. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.

Any idea what I am missing?

**Comment by ravensorb** [comment]
Ok, it looks like the latest build "NuProj.0.10.37-beta-gc300461e6d\tools" - fixes the issue.  Any chance you could publish this version as a public nuget package?

**Comment by AArnott** [comment]
That would require @terrajobst, who hasn't spent much time on this project lately.
But I've been pushing package updates fairly regularly to my feed on myget.org if you want to add that as a package source.

Feed: https://www.myget.org/F/aarnott/api/v3/index.json
Gallery: http://myget.org/gallery/aarnott

**Comment by ravensorb** [comment]
Thanks however that would require everyone that contributes on a few of my projects to add that as a custom feed -- something I was hoping to avoid :)  Wonder if @terrajobst would add you as a co-owner to the nuget package?

**Comment by AArnott** [comment]
Not at all. I strongly recommend you have a nuget.config file in the root of your repo anyway. That way, your build will always work regardless of how any individual machine is configured. 

Here's [a sample](https://github.com/AArnott/pinvoke/blob/master/src/nuget.config).

**Comment by ravensorb** [comment]
Hmm, I'll have to see if this will cause an issue internally :)  Thanks.  

Note: It would still be great if we could get the main one updated or at least de-listed.

**Comment by AArnott** [comment]
> It would still be great if we could get the main one updated or at least de-listed.

I *so* agree. 
In fact now that I've written scripts that can automatically push packages to nuget.org when pushing tags to github.com for anyone with push privileges, which can be done without exposing the nuget.org account password to anyone, I hope @terrajobst can set that up so other maintainers can update whenever it's appropriate.
